client/src/components/WorkflowInvocationState/Export/BioComputeObjectExportCard.vue

```diff
@@ -0,0 +1,174 @@
+                            <input
+                                id="authorization"
+                                v-model="form.authorization"
+                                type="password"
+                                class="form-control"
+                                required />
+                        </div>
+                        <div class="form-group">
+                            <label for="table">Prefix</label>
+                            <input
+                                id="table"
+                                v-model="form.table"
+                                type="text"
+                                class="form-control"
+                                placeholder="BCO"
+                                required />
+                        </div>
+                        <div class="form-group">
+                            <label for="owner_group">User Name</label>
+                            <input
+                                id="owner_group"
+                                v-model="form.owner_group"
+                                type="text"
+                                class="form-control"
+                                required />
+                        </div>
+                        <div class="form-group">
+                            <button class="btn btn-primary">{{ "Submit" | localize }}</button>
+                        </div>
+                    </form>
+                </div>
+            </b-tab>
+        </b-tabs>
+    </div>
+</template>
+
+<script>
+import { getRootFromIndexLink } from "onload";
+import { getAppRoot } from "onload/loadConfig";
+import { mapGetters } from "vuex";
+import axios from "axios";
+
+const getUrl = (path) => getRootFromIndexLink() + path;
+
+export default {
+    props: {
+        invocationId: {
+            type: String,
+            required: true,
+        },
+    },
+    data() {
+        return {
+            bco: {
+                type: Object,
+                required: true,
+            },
+            form: {
+                fetch: "",
+                authorization: "",
+                table: "",
+                owner_group: "",
+            },
+        };
+    },
+    computed: {
+        ...mapGetters(["getInvocationById"]),
+        indexStr() {
+            if (this.index == null) {
+                return 0;
+            } else {
+                return `${this.index + 1}`;
+            }
+        },
+        bcoDownloadLink: function () {
+            return `${getAppRoot()}api/invocations/${this.invocationId}/biocompute/download`;
+        },
+    },
+    created: function () {
+        axios
+            .get(getUrl(`./api/invocations/${this.invocationId}/biocompute/`))
+            .then((response) => {
+                this.bco = response.data;
+            })
+            .catch((e) => {
+                this.errors.push(e);
+            });
+    },
```

Comment:
I think this request should be made directly in the `submitForm` method, otherwise, whenever you expand a random workflow invocation from the list the request will be made even if you don't want to export anything.

Reply:
Resolved with [1bcd5fa](https://github.com/galaxyproject/galaxy/pull/14374/commits/1bcd5fac4a09b7e2b3a92e32cdb0403877fb0453)